### PR TITLE
Ft rework login page 157692939

### PR DESCRIPTION
**Pull request by akash-011**
#### What does this PR do?
Applies new Landing page designs 
#### Where should the reviewer start?
FIle changes in PR
#### How should this be manually tested?
Run the application and look at login page
#### What are the relevant pivotal tracker stories?
[#157692939](https://www.pivotaltracker.com/story/show/157692939)
#### Screenshots (if appropriate)
N/A

### CODE REVIEW

File: src/pages/Login/LoginPage.scss [hunk 1]
```diff
@@ -1,59 +1,93 @@
...
         }
+            .underline{
+                position: absolute;
+                background-color: #fdb237;
+                width: 8vw;
+                height: 0.5vh;
+                border-radius: 5px;
+                left: 68vw;
+                top: 39vh;
+
```
Comment (jimmykimani) [inline src/pages/Login/LoginPage.scss]: Remove the unnecessary spaces

Comment (jimmykimani) [reply src/pages/Login/LoginPage.scss]: Also, Indent your code your scss format is incorrect  lets follow the style guidlines here is the link https://sass-guidelin.es/#syntax--formatting

File: src/pages/Login/LoginPage.scss [hunk 2]
```diff
@@ -1,59 +1,93 @@
...
-            .instructions {
-                margin: 0.2rem;
+                .wire{
+                    font-family: 'DIN Pro Light';
+                    font-style: italic;
+                }
+                
             }
+        }
+        .button{
```
Comment (jimmykimani) [inline src/pages/Login/LoginPage.scss]: Let's add a border radius to the button to resemble the mockup

Comment (akash-011) [reply src/pages/Login/LoginPage.scss]: I tried that , it didnt work with the raised button component, is there another way or should be change it to a normal button ?

File: src/pages/Login/LoginPage.scss (hunk 1 above)
Comment (akash-011) [reply src/pages/Login/LoginPage.scss]: Cool , I shall sort that out

**Comment by kimobrian** [comment]
image

@jimmykimani @akash-011 Is this what we are expecting. Did you upload the logo image?

**Comment by akash-011** [comment]
@kimobrian The relevant image has been uploaded

**Pull request merged**